test/dotnet.Tests/dotnet.Tests.csproj

```diff
@@ -57,6 +57,7 @@
     <Compile Include="..\Microsoft.DotNet.Configurer.UnitTests\**\*.cs" LinkBase="Configurer" />
     <Compile Include="..\Microsoft.DotNet.ShellShim.Tests\**\*.cs" LinkBase="ShellShim" />
     <Compile Remove="..\dotnet-test.Tests\GivenDotnetTestBuildsAndRunsHelp.cs" />
+    <Compile Remove="..\dotnet-test.Tests\MSBuildHandlerTests.cs" />
```

Comment:
Why were the MSBuildHandlerTests.cs tests removed? I'm cleaning up the dotnet.Tests project and related test folder structure.

Reply:
https://github.com/dotnet/sdk/pull/48310 let's see if that builds.